### DISCUSSION TITLE
Pressure test

### DISCUSSION THREAD

**Discussion by harryGuo133** [uNetworking/uWebSockets#1749] [Q&A] [answered]
wsPtr->getBufferedAmount() ,
Using this function, you find that you get larger and larger values, And it hasn't decreased. Why?
Does this function interface get the cache of the socket or the cache of the library? What size is it exactly?

**Comment by uNetworkingAB** [comment, marked as answer]
You can study this example https://github.com/uNetworking/uWebSockets.js/blob/master/examples/Backpressure.js

**Reply by harryGuo133** [reply]
thx  for your reply.
